### ISSUE TITLE
Feature request - Call entity damage event when storm staff causes lightning damage

### ISSUE THREAD

**Issue by ChanceSD** [Slimefun/Slimefun4#3408]
### ❗ Checklist

- [X] I am using the official english version of Slimefun and did not modify the jar.
- [X] I am using an up to date "DEV" (not "RC") version of Slimefun.
- [X] I am aware that issues related to Slimefun addons need to be reported on their bug trackers and not here.
- [X] I searched for similar open issues and could not find an existing bug report on this.

### 📍 Description

As requested by @Sfiguz7 
Basically to have some way for other plugins to identify that lightning from storm staff came from a player so it can be blocked as PvP. Suggestions were metadata or calling the damage event.

### 📑 Reproduction Steps

N/a

### 💡 Expected Behavior

N/a

### 💻 Server Software

Spigot

### 🎮 Minecraft Version

1.18.x

### ⭐ Slimefun version

N/a

**Comment by TheBusyBiscuit** [comment]
@Sfiguz7 You very well know feature requests do not belong on the bug tracker...

**Comment by Sfiguz7** [comment]
I do but we don't really have a place except here. I don't think this should be made a suggestion, it is not exactly an addition just a compatibility fix that would make Slimefun's accessibility better overall, and I still wanted to have it written somewhere to reference when PRing

**Comment by variananora** [comment]
A little off topic but I think this could 'fix' #3396

**Comment by TheBusyBiscuit** [comment]
Closing this issue as it's not really a bug report and its already served its purpose of reminding @Sfiguz7 to make a PR.
The discussion can continue in #3411 if there is a need for further discussion in the first place.
